scala-package/examples/src/main/scala/ml/dmlc/mxnetexamples/inferexample/SSDClassifierExample.scala

```diff
@@ -0,0 +1,96 @@
+package SSDClassifierExample
+
+import ml.dmlc.mxnet._
+import ml.dmlc.mxnet.infer._
+import org.kohsuke.args4j.{CmdLineParser, Option}
+import org.slf4j.LoggerFactory
+
+import scala.collection.JavaConverters._
+import java.nio.file.{Files, Paths}
+
+class SSDClassifierExample {
+  @Option(name = "--model-dir", usage = "the input model directory")
+  private val modelPath: String = "/model"
+  @Option(name = "--model-prefix", usage = "The prefix of the model")
+  private val modelPrefix: String = "/ssd_resnet50_512"
+  @Option(name = "--input-image", usage = "the input image")
+  private val inputImagePath: String = "/images/Cat-hd-wallpapers.jpg"
```

Comment:
can we stick to kitten.jpg since it's probably around in the other test artifacts?